agent/consul/internal_endpoint.go

```diff
@@ -188,6 +188,49 @@ func (m *Internal) ServiceTopology(args *structs.ServiceSpecificRequest, reply *
 		})
 }
 
+// IntentionUpstreams returns the upstreams or downstreams of a service. Upstreams and downstreams are inferred from intentions.
```

Comment:
> IntentionUpstreams returns the upstreams or downstreams

Why does it return both if it's named for only upstreams?

Reply:
Fixed. It will return both in the future, but I updated the comment to reflect its current state.